meta-lmp-base/conf/layer.conf

```diff
@@ -35,6 +35,13 @@ BBFILES_DYNAMIC += " \
 # A list of recipes that are completely stable and will never change.
 SIGGEN_EXCLUDERECIPES_ABISAFE += "os-release"
 
+# A list of recipe dependencies that should not be used to determine signatures
+# of tasks from one recipe when they depend on tasks from another recipe.
+SIGGEN_EXCLUDE_SAFE_RECIPE_DEPS += " \
+    initramfs-ostree-lmp-image->os-release \
+    core-image-minimal-initramfs->os-release \
```

Comment:
Can you also add initramfs-ostree-lmp-recovery here?

Reply:
The initramfs-ostree-lmp-recovery explicitly install the os-release so it should rebuild.

Reply:
You are right, forgot about that one.